src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt

```diff
@@ -31,7 +31,8 @@ data class GrpcServiceConfiguration(
     @Deprecated("For removal since v3.37") @JsonProperty(required = true) var strategy: RoutingStrategy<*>,
     @JsonProperty(required = true, value = "service-class") var serviceClass: Class<*>,
     @JsonProperty(required = true) var endpoints: Map<String, GrpcEndpointConfiguration> = emptyMap(),
-    @JsonProperty var filters: List<Filter> = emptyList()
+    @JsonProperty var filters: List<Filter> = emptyList(),
+    @JsonProperty(required = false) var keepAliveInterval: Long = 300L
```

Comment:
Let's move this property to GrpcConfiguration.client.keepAliveInterval

Reply:
Done